lib/test_printers.ml

```diff
@@ -38,11 +38,6 @@ let option : 'a printer -> 'a option printer = fun pr x ->
 	| None -> "None"
 	| Some s -> pr s |> Printf.sprintf "Some %s"
 
-let either : 'a printer -> 'b printer -> ('a, 'b) Xapi_stdext_monadic.Either.t printer = fun pr_a pr_b x ->
```

Comment:
Depends on https://github.com/xapi-project/xcp-networkd/pull/178, which needs to be merged first, that removes the usage of `either`.